src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -261,6 +262,61 @@ private static void AddRequiresLicenseAcceptance(Document document, IDictionary<
             }
         }
 
+        private static void AddPackageTypes(Document document, IDictionary<string, string> package)
+        {
+            string flattenedPackageTypes;
+            StringBuilder searchIndex = new StringBuilder();
+            if (package.TryGetValue("flattenedPackageTypes", out flattenedPackageTypes))
```

Comment:
This was to have consistency with a flattened format that the dependencies also seemed to use, and was for the V2 API. As we discussed in the other thread, I'm actually okay with getting rid of this completely if we don't need this in the V2 API.

---
In reply to: [75567354](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r75567354) [](ancestors = 75567354)